src/services/utilities.ts

```diff
@@ -437,13 +437,53 @@ namespace ts {
     }
 
     /**
+     * returns true if the position is in between the open and close elements of an JSX expression.
+     */
+    export function isInsideJsxElementOrAttribute(sourceFile: SourceFile, position: number) {
+        let token = getTokenAtPosition(sourceFile, position);
+
+        if (!token) {
+            return false;
+        }
+
+        // <div>Hello |</div>
+        if (token.kind === SyntaxKind.LessThanToken && token.parent.kind === SyntaxKind.JsxText) {

```

Comment:
Still doesn't make sense to me that the token is considered a child of the JsxText. That should be the child of the closing JSX tag. So I can't imagine how you're hitting this.